app/support/reservations/validations.rb

```diff
@@ -23,12 +23,12 @@ module Reservations::Validations
     validate :duration_is_interval
 
     validates :category,
-              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: false },
+              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: true },
               if: -> (r) { r.class.const_defined?(:CATEGORIES) }
 
     validates :category,
               absence: true,
-              unless: -> (r) { r.class.const_defined?(:CATEGORIES) }
+              unless: -> (r) { r.class.const_defined?(:CATEGORIES) || r.admin? }
```

Comment:
I'd like @drench to weigh in on this. I think the inheritance of these validations  I would vote from removing the `absence` is a little funky. I would vote for moving the `inclusion` off of the superclass and define them in `AdminReservation` and `OfflineReservation` (I think the `allow_blank` logic is different between the two). And I'm not really sure the `absence` validation actually adds any value. Could we remove this validation altogether?